metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/api/internal/ReferenceTerm.java

```diff
@@ -2,11 +2,32 @@
 
 import eu.europeana.enrichment.utils.EntityType;
 import java.net.URL;
-import java.util.List;
+import java.util.Set;
 
+/**
+ * Represents a reference term: a URL reference in a record with a list of candidate reference
+ * types. Two reference terms are equal if and only if they have the same reference and candidate
+ * types.
+ */
 public interface ReferenceTerm {
 
-  List<EntityType> getCandidateTypes();
+  Set<EntityType> getCandidateTypes();
 
   URL getReference();
+
```

Comment:
Same here 

"Equals and hashCode shouldn't be in the interface imo.
The implementer should decide if the equals should be applied on the implementing class or not.
Btw they were removed on a previous review."